.github/workflows/static-analysis.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          version: v1.53.3
+          version: v1.54.1
```

Comment:
This can be merged separately.

Reply:
I think it should be kept to prevent new contributors
from using the new features of 1.21.